### DISCUSSION TITLE
display more than 10 slides in the Image Carousel Widget

### DISCUSSION THREAD

**Discussion by yaron86** [elementor/elementor#29061] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

solve me to get my layout that i need 

### Describe the solution you'd like.

in the Image Carousel Widget i want to be able to choose "custom" and choose the Slide Number to show. 
![464784859_10162711433239752_9137730106103444903_n](https://github.com/user-attachments/assets/3c2a0f2e-d28a-4fb9-8fd4-d1aaf692f0be)

### Additional context

for now maybe you guys have a custom code to provide to set a 11 items to show 

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by manthankanani** [comment]
If you want it without requesting feature in urgency.
Right now you can do it by inspect element and option as you want between
<select></select> tag and sav.

**Comment by hitsgee** [comment]
I think this is performance related. I tend to stay around 4-5 slides max depending on layout required.
